packages/@holaplexui-react/src/components/Form.tsx

```diff
@@ -302,6 +302,48 @@ const FormCheckbox = forwardRef(function FormCheckbox(
 
 Form.Checkbox = FormCheckbox;
 
+interface FormRadioButtonGroupProps
+  extends DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement> {
+  items: { label: string; value: string; id: string }[];
+  groupName: string;
+  onOptionChange?: () => void;
+  className?: string;
+}
+
+const FormRadioButtonGroup = forwardRef(function FormRadioButtonGroup(
+  { items, groupName, onOptionChange, className, ...props }: FormRadioButtonGroupProps,
+  ref
+) {
```

Comment:
We should always try to conform to the props react-form-hook is going to provide like onChange, name, and value. Not groupName and onOptionChange.